crates/bitwarden/src/vault/mod.rs

```diff
@@ -0,0 +1,3 @@
+mod folder;
+
+pub use folder::*;
```

Comment:
@MGibson1 had some hesitation against re-exporting wildcard since it's not trivial to know what gets re-exported.

Reply:
I can agree with that yeah, specially for `pub use` I'd rather be specific, I've removed the wildcard now.